src/workloads/scale/BigUpdate.yml

```diff
@@ -97,7 +97,7 @@ Actors:
     Threads: 100
     Database: *DB
     CollectionCount: 10  # This is specifically 1% of collections
-    UpdateFilter: {y: {^RandomInt: {distribution: poisson, mean: 100}}}
+    UpdateFilter: {y: {^RandomInt: {distribution: poisson, mean: 100.0}}}
```

Comment:
After making the changes from 4f4e69b27d94446e8056c4bc996ff09b233c4148 you should be able to revert the changes to this file as well.